### PR TITLE
Feature/framwework structure

### PR DESCRIPTION
**Pull request by antho1404**

### CODE REVIEW

File: .vuepress/config.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            items: [
+              { text: 'Forum', link: 'https://forum.mesg.com' },
+              { text: 'Discord', link: 'https://discord.gg/5tVTHJC' }
+            ]
+          },
+          {
+            text: 'Libraries',
+            items: [
+              { text: 'NodeJS', link: 'https://github.com/mesg-foundation/js-sdk' },
+              { text: 'Golang (Service)', link: 'https://godoc.org/github.com/mesg-foundation/engine/client/service' },
```
Comment (NicolasMahe) [inline .vuepress/config.js]: doesn't exist anymore

File: .vuepress/config.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+              { text: 'C# (Not official)', link: 'https://github.com/neville-nazerane/mesg-apis' }
```
Comment (NicolasMahe) [inline .vuepress/config.js]: not updated since 1 year

File: .vuepress/config.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        items: [
+          {
+            text: 'Help',
+            items: [
+              { text: 'Forum', link: 'https://forum.mesg.com' },
+              { text: 'Discord', link: 'https://discord.gg/5tVTHJC' }
+            ]
+          },
+          {
+            text: 'Libraries',
```
Comment (NicolasMahe) [inline .vuepress/config.js]: should we just remove this links?

**Comment by antho1404** [comment]
closing in favor of https://github.com/liteflow-labs/docs/pull/1

**Pull request closed without merging**